statsmodels/tsa/tests/test_holtwinters.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
```

Comment:
Yeah I realised once the tests had been run, I was going to have to change a few things.
Trying to figure out what the lowest common denominator will be to get this all working.